### PR TITLE
Restore condition for project-specific emissions reporting

### PR DESCRIPTION
**Pull request by mellamoSimon**
This restores a condition designed to enable the reporting of industry emissions from feedstocks with an ad-hoc ex-post fix designed for the Ariadne project. Used when feedstock flows are not available (older remind versions).
This PR is NOT intended to fix the inconsistencies in emissions accounting. That will come in this one: https://github.com/pik-piam/remind2/pull/530 .

### CODE REVIEW

**Comment by fbenke-pik** [comment]
The test also fails for ECEMF. The problem is the variable `emi.vars.wNonEn`, which is declared under the condition you introduced, is also used later on in line 2770: https://github.com/mellamoSimon/remind2/blob/fix-emissions-inconsistency/R/reportEmi.R#L2770

So not knowing anything about the specifics of this reporting, I can at least tell you that you would have to make sure that  `emi.vars.wNonEn` is defined with reasonable variables under any condition.

**Comment by mellamoSimon** [comment]
ok, will do, thank you! For the record: I didn't realize that the mapping tests can be skipped and buildLibrary still is successful when running locally (which is what had happened here)

File: R/reportEmi.R [hunk 1]
```diff
@@ -496,19 +494,20 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
...
 
  # CO2 emissions from the end-of-life of carbon-bearing products
+ # fixme: wild guess: mybe I have to discount the carbon of biogenic and atmospheric origin from here as well. Nope, wasn't that
 if (!is.null(vm_plasticsCarbon)) {
     out <- mbind(out,
-               setNames(dimSums(vm_feedstockEmiUnknownFate, dim=3)* GtC_2_MtCO2,
+               setNames((dimSums(vm_feedstockEmiUnknownFate, dim=3))* GtC_2_MtCO2,
                         "Emi|CO2|Energy|Waste|+|Feedstocks unknown fate (Mt CO2/yr)"),
-               setNames(dimSums(vm_incinerationEmi, dim=3)* GtC_2_MtCO2,
+               setNames((dimSums(vm_incinerationEmi, dim=3))* GtC_2_MtCO2,
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline R/reportEmi.R:510]: What is up with those brackets?

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]

**Comment by mellamoSimon** [comment]
I ran all the mapping tests manually and they all passed (with warnings of missing variables for some and the gaps in the summation checks, as expected)

File: R/reportEmi.R (hunk 1 above)
Comment (mellamoSimon) [reply R/reportEmi.R]: ups, I was trying something and forgot to delete those. Now I did.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
:shipit:

**Pull request merged**